PlugMe-ng/plugme-web-ui
# Load notifications on the same tab

#### What does this PR do?
Load notifications and messages on the same tab
#### Description of Task to be completed?
Currently, a new message/notification opens up in a new tab, the objectives on this task is show everything on the same tab. 
#### How should this be manually tested?
- Go to `https://plugme-client-staging.herokuapp.com/`
- Check if you have notifications
- Then click on any of your notifications,
- it should load the corresponding page without opening a new tab. 

#### Any background context you want to provide?
N/A
#### What are the relevant pivotal tracker stories?
N/A
#### Screenshots (if appropriate)
N/A
